website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -7282,7 +7282,16 @@ To create an SDK wrapper for use in a Python pipeline, do the following:
 
 #### 13.1.2. Creating cross-language Python transforms
 
-To make your Python transform usable with different SDK languages, you must create a Python module that registers an existing Python transform as a cross-language transform for use with the Python expansion service and calls into that existing transform to perform its intended operation.
+Any Python transforms defined in the scope of the expansion service should be accessible by specifying their fully qualified names. For example, you could use Python's `ReadFromText` transform in a Java pipeline with its fully qualified name `apache_beam.io.ReadFromText`:
+
+```java
```

Comment:
Can we just add Go snippets to also cover this part for Go SDK or will it need other documentation changes ?

cc: @riteshghorse

Reply:
I think it deserves to be a separate PR.

Reply:
Ok, I'll track it in a separate issue. Thanks!